src/feedback/trulens/feedback/v2/feedback.py

```diff
@@ -254,6 +254,17 @@ class Abstention(Semantics, WithPrompt):
     )
 
 
+class Trivial(Semantics, WithPrompt):
+    system_prompt: ClassVar[str] = cleandoc(
+        """Consider the following list of statements. Remove any statements that are trivial, irrelevant, or unimportant. Respond with a list of the remaining statements."""
+    )
```

Comment:
We did not clearly define in this prompt, what a trivial statement is...

Reply:
suggestion 
```
"Identify and remove sentences that are stylistic, contain trivial pleasantries, or lack substantive information relevant to the main content."
```

Credits to ChatGPT